src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -154,11 +157,13 @@ const constructPageFetchUrl = ({
 
   if (isLocal) {
     switch (pageType) {
-      case ARTICLE_PAGE:
-        fetchUrl = Url(
-          `/${service}/articles/${id}${variant ? `/${variant}` : ''}`,
-        );
-        break;
+      case ARTICLE_PAGE: {
+        const isCpsId = isCpsIdCheck(id);
+        if (isCpsId) {
+          return Url(`/${id}`);
+        }
+        return Url(`/${service}/articles/${id}${variant ? `/${variant}` : ''}`);
+      }
```

Comment:
LGTM
Shall we only perform assignment in this block to retain the `switch...case` format since
the value of `fetchUrl` is returned [here](https://github.com/bbc/simorgh/pull/11529/files#diff-b0291bb91a24efe1af51dd8467d6e14fa879769816d63e1fb5f2622819350b90R186)?